### PR TITLE
Fixes links to metapackages / removes mention of new Venia sample data repo

### PR DESCRIPTION
**Pull request by bdenham**
## Description

**UPDATE:** 
Removed all private links that result in 404s, including the Jira links which are now commented out in the description. That was an experiment that I now think better of in favor of no 404 links from the changelog. 

**NOTE**
I changed the urls for the the venia sample data repo from https://github.com/magento-commerce/venia-sample-data-modules to the currently non-existing repo — https://github.com/magento/venia-sample-data-modules. To be clear, PWA community members cannot access the current sample data repo: https://github.com/magento-commerce/venia-sample-data-modules. We must create a public repo in our GitHub `magento` organization.

This PR fixes the links to the internal `magento-commerce` repos for the new PWA Studio metapackages. It also removes or notes that the new Venia Sample Data modules repo will not be available for access until the next release. Maybe. 

## Related Issue

NA - New links have been created for our new public PWA Studio metapackage repos.

Closes JIRA [PWA-2394](https://jira.corp.magento.com/browse/PWA-2394)

### Verification Stakeholders

- @dpatil-magento 
- @davemacaulay 
- @supernova-at

## Verification Steps

Make sure the new links work.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
...
+-  **Added PWA Studio metapackages** — In this release, we introduce our PWA Studio metapackages: One for [Magento Open Source](https://github.com/magento/magento2-pwa) and one for [Adobe Commerce](https://github.com/magento/magento2-pwa-commerce). These metapackages provide a new way to add any additional Open Source or Commerce features needed by your PWA modules. The PWA Studio team will also use these metapackages to add new features to the Open Source and Commerce code bases as needed. In fact, we did that in this release. We added GraphQL endpoints and fields to the metapackages to fix some `Cart`-related bugs. GitHub PRs: [4][], [1][].
```
Comment (keharper) [inline CHANGELOG.md:9]:
```suggestion
-  **Added PWA Studio metapackages** — In this release, we introduce our PWA Studio metapackages: One for [Magento Open Source](https://github.com/magento/magento2-pwa) and one for [Adobe Commerce](https://github.com/magento/magento2-pwa-commerce). These metapackages provide a new way to add any additional Open Source or Commerce features needed by your PWA modules. The PWA Studio team will also use these metapackages to add new features to the Open Source and Commerce code bases as needed. In fact, we did that in this release. We added a GraphQL mutation and several fields to the metapackages to fix some `Cart`-related bugs. GitHub PRs: [4][], [1][].
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
...
 
 -  **Added new PWA Tailwind theming to Venia Header** — Refactored the Venia site `Header` component to use our new [Tailwind](https://tailwindcss.com/) theming framework. The `Header` component is the first component to use our theming framework. Other components will follow in the coming releases. GitHub PR: [3472][].
 
 -  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento-commerce/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
 
--  **Added new GraphQL endpoint for Contact and Newsletter forms** — You can now use GraphQL to submit Contact Us and Newsletter form data to the Commerce backend. From there, you can use it to send emails and personalize other customer interactions. GitHub PR: [5][].
+-  **Added a GraphQL mutation for submitting the Contact Us form.** — You can now use GraphQL to submit the Contact Us form data to the Open Source or Commerce backend. From there, you can use it to send emails and personalize other customer interactions.
+
```
Comment (keharper) [inline CHANGELOG.md:22]: Are you sure about the last sentence? The mutation just sends a Contact message to the merchant. Although I'm sure you can style the form any way you want, I don't foresee the ability to do anything else with it. If you want to develop a totally different form to contact the merchant, you'd have to develop a custom module.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
...
+-  **Added GraphQL `storeConfig` fields** — Use this field to determine whether the Contact Us and Newletter features are enabled.
```
Comment (keharper) [inline CHANGELOG.md:23]:
```suggestion
-  **Added GraphQL `storeConfig` fields** — Use these fields to determine whether the Contact Us and Newletter features are enabled.
```

File: CHANGELOG.md (hunk 2 above)
Comment (bdenham) [reply CHANGELOG.md:22]: Yep, you're right. This was just a "see what kind of things you can do with this" sentence, which could be a variety of things. Good call.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
...
 
--  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento-commerce/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
+-  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
 
--  **Added new GraphQL endpoint for Contact and Newsletter forms** — You can now use GraphQL to submit Contact Us and Newsletter form data to the Commerce backend. From there, you can use it to send emails and personalize other customer interactions. GitHub PR: [5][].
+-  **Added a GraphQL mutation for submitting the Contact Us form.** — You can now use GraphQL to submit the Contact Us form data to the Open Source or Commerce backend.
```
Comment (keharper) [inline CHANGELOG.md:21]:
```suggestion
-  **Added a GraphQL mutation for submitting the Contact Us form.** — You can now use the `contactUs` mutation to submit the Contact Us form data to the Open Source or Commerce backend.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
...
+
+-  **Added GraphQL `storeConfig` fields** — Use these fields to determine whether the Contact Us and Newletter features are enabled.
```
Comment (keharper) [inline CHANGELOG.md:23]:
```suggestion
-  **Added GraphQL `storeConfig` fields** — Use the `contact_enabled` and `newsletter_enabled` fields in a `storeConfig` query to determine whether the Contact Us and Newsletter features are enabled.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
...
+-  **Added PWA Studio metapackages** — In this release, we introduce our PWA Studio metapackages: One for [Magento Open Source](https://github.com/magento/magento2-pwa) and one for [Adobe Commerce](https://github.com/magento/magento2-pwa-commerce). These metapackages provide a new way to add any additional Open Source or Commerce features needed by your PWA modules. The PWA Studio team will also use these metapackages to add new features to the Open Source and Commerce code bases as needed. In fact, we did that in this release. We added a GraphQL mutation and several fields to the metapackages to fix some `Cart`-related bugs. GitHub PRs: [4][], [1][].
```
Comment (keharper) [inline CHANGELOG.md:9]: Sorry about the churn. Change the last sentence to say something like

We extended the GraphQL schema to include a new mutation and to provide more details when a cart item error occurs.

Currently, you don't provide any details about https://github.com/magento-untitled-geese/magento2-pwa/pull/2/files, so being vague at the top lets you dive into detail below without repeating yourself.

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
...
 
--  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento-commerce/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
+-  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
 
--  **Added new GraphQL endpoint for Contact and Newsletter forms** — You can now use GraphQL to submit Contact Us and Newsletter form data to the Commerce backend. From there, you can use it to send emails and personalize other customer interactions. GitHub PR: [5][].
+-  **Added a GraphQL mutation for submitting the Contact Us form.** — You can now use GraphQL to submit the Contact Us form data to the Open Source or Commerce backend.
+
+-  **Added GraphQL `storeConfig` fields** — Use these fields to determine whether the Contact Us and Newletter features are enabled.
```
Comment (keharper) [inline CHANGELOG.md:24]: New item:

-  **Added GraphQL schema to expose the error status of cart items. The `CartItemInterface` now contains the `errors` field, which uses the `CartItemError` data type to return an error code an message.

Comment (bdenham) [reply CHANGELOG.md:24]: Excellent

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,38 +6,42 @@ _For older release notes, see_ [PWA Studio releases][].
...
 
-    **For Adobe Commerce backends**: Install the [Adobe Commerce metapackage](https://github.com/magento-commerce/magento2-pwa-commerce).
+    **For Adobe Commerce backends**: Install the [Adobe Commerce metapackage](https://github.com/magento/magento2-pwa-commerce).
 
 -  **Added new PWA Tailwind theming to Venia Header** — Refactored the Venia site `Header` component to use our new [Tailwind](https://tailwindcss.com/) theming framework. The `Header` component is the first component to use our theming framework. Other components will follow in the coming releases. GitHub PR: [3472][].
 
--  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento-commerce/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
+-  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
```
Comment (supernova-at) [inline CHANGELOG.md:19]: Based on Slack conversations, we're going to drop this bullet point.

File: CHANGELOG.md [hunk 9]
```diff
@@ -6,38 +6,42 @@ _For older release notes, see_ [PWA Studio releases][].
...
 
     **UPDATES REQUIRED!** This release and all future PWA Studio releases will have dependencies on these metapackages. That means you need to add one or both of the metapackages to your PWA apps (depending on your backend target). Instructions for installing these packages are in the README files for each metapackage repo:
 
-    **For Open Source backends**: Install the [Magento Open Source metapackage](https://github.com/magento-commerce/magento2-pwa).
+    **For Open Source backends**: Install the [Magento Open Source metapackage](https://github.com/magento/magento2-pwa).
 
-    **For Adobe Commerce backends**: Install the [Adobe Commerce metapackage](https://github.com/magento-commerce/magento2-pwa-commerce).
+    **For Adobe Commerce backends**: Install the [Adobe Commerce metapackage](https://github.com/magento/magento2-pwa-commerce).
```
Comment (supernova-at) [inline CHANGELOG.md:15]: I thought that we were going to host instructions on how to install this via composer on the docs site and then link to that article from here?

I'm not sure we want this repository to be open source / publicly available.

Comment (bdenham) [reply CHANGELOG.md:15]: We are. See https://github.com/AdobeDocs/commerce-pwa-studio/pull/75

**Comment by bdenham** [comment]
@supernova-at: Updated links to point to docs and removed references to venia sample data.

File: CHANGELOG.md (hunk 8 above)
Comment (bdenham) [reply CHANGELOG.md]: Done

**Review by davemacaulay — APPROVED** [review]
Looks good, the links will need to resolve properly though!

**Review by supernova-at — DISMISSED** [review]

**Review by supernova-at — DISMISSED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -6,38 +6,38 @@ _For older release notes, see_ [PWA Studio releases][].
...
 
--  **Added new PWA Tailwind theming to Venia Header** — Refactored the Venia site `Header` component to use our new [Tailwind](https://tailwindcss.com/) theming framework. The `Header` component is the first component to use our theming framework. Other components will follow in the coming releases. GitHub PR: [3472][].
+-  **Added new PWA Tailwind theming to Venia Header** — Refactored the Venia site `Header` component to use our new [Tailwind](https://tailwindcss.com/) theming framework. The `Header` component is the first component to use our theming framework. Other components will follow in the coming releases.
 
--  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento-commerce/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
+-  **Added a GraphQL mutation for submitting the Contact Us form.** — You can now use the `contactUs` mutation to submit the Contact Us form data to the Open Source or Commerce backend.
```
[the remark this answers is not in the document]
Comment (bdenham) [reply CHANGELOG.md:19]: Asking @eug123.

[the remark this answers is not in the document]
Comment (bdenham) [reply CHANGELOG.md:19]: @keharper — @eug123 confirmed this change has not been published yet.

**Review by supernova-at — APPROVED** [review]

**Review by davemacaulay — APPROVED** [review]

**Pull request merged**